airgun/views/host.py

```diff
@@ -297,3 +366,134 @@ class HostsChangeEnvironment(BaseLoggedInView):
     def is_displayed(self):
         return self.browser.wait_for_element(
             self.title, exception=False) is not None
+
+
+class OnTaxonomyMismatchCheckBoxGroup(RadioGroup):
```

Comment:
so I am still prefer to change that approach on a single:
class OnTaxonomyMismatchCheckBoxGroup(View):
    fix_location = Text/Checkbox/whatever(id='location_optimistic_import_yes')
    fail_on_mismatch = Text/Checkbox/whatever(id='location_optimistic_import_no')
```
and use it in the same way as here through `fill` and `read`
Proposed solution is not bad, but I don't want to populate extra code which give us not so much of benefit and especially when it covers wrong scenarios